sheets/writer.go

```diff
@@ -32,14 +32,16 @@ import (
 const insertDataOption = "INSERT_ROWS"
 
 type Writer struct {
-	sheets           *sheets.Service
-	sheetName        string
-	spreadsheetID    string
+	sheets        *sheets.Service
+	sheetName     string
+	spreadsheetID string
+	// haris: we should explain what this is
 	valueInputOption string
 	maxRetries       uint64
 	retryCount       uint64
 }
 
+// haris: what if valueInputOption is an invalid option?
```

Comment:
It is validated in config Parse, but if it is invalid, sheets client will return error

Reply:
Even though valueInputOption is kept an an optional parameter, so even if the user skips the default value 'USER_ENTERED' is assigned. Yes, we can make it as a constant, to eliminate the scope of any error.

Reply:
Gotcha. We can do that separately IMHO.